sdk/javascript/src/nem/external/tweetnacl-nacl-fast-keccak.js

```diff
@@ -12,9 +12,9 @@ const { keccak_512 } = require('js-sha3');
 // See for details: http://tweetnacl.cr.yp.to/
 
```

Comment:
as mentioned, I would not touch this file at all...

Reply:
i think it's low risk (most diffs can ignore whitespace) and we did same to external one in py 😅

Reply:
😬